Create ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md (#2163)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1018-503-96-device-data-widget-with-long-software-list-fix-6475-graft-release.md

---
date: ""

> title: Device data widget with long software list fix (#6475) [GRAFT][release/y2024] (#6500)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fixed an issue with the device data widget when displaying a long software list
```

> title: Device data widget with long software list fix (#6475) [GRAFT][release/y2024] (#6500)

Reviewer: ```suggestion
title: The device data widget with a long software list is fixed by transforming the software list into a paginated one.
```

> title: The device data widget with a long software list is fixed by transforming the software list into a paginated one.

Reviewer 2: ```suggestion
title: Software list in Device data widget now paginated to show all entries properly
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3669
version: 1018.503.96
---

> Device data widget with long software list fix (#6475) [GRAFT][release/y2024] (#6500)

Author: The Claude AI suggested to use the following body:
```suggestion
In the device data widget, the software list was not displayed correctly if it contained a large number of entries, making it difficult for users to view and manage the installed software on their devices. This issue has now been resolved. The device data widget now properly displays long software lists, ensuring that users have full visibility of all installed software, regardless of the number of entries. This improvement enhances the usability and functionality of the device management capabilities.
```

> Device data widget with long software list fix (#6475) [GRAFT][release/y2024] (#6500)

Reviewer: ```suggestion
In the device data widget, the software list was not displayed correctly if it contained a large number of entries. This issue has now been resolved by restricting the initial software list by only 10 items and an option to load more if necessary.
```

> In the device data widget, the software list was not displayed correctly if it contained a large number of entries. This issue has now been resolved by restricting the initial software list by only 10 items and an option to load more if necessary.

Reviewer 2: ```suggestion
In the "Device data" widget, the software list was not displayed correctly if it contained a large number of entries. This issue has now been resolved by paginating the software list, that is, limiting the initial list to 10 items and an option to load more if needed.
```
